docs/learn/betting_strategy.md

```diff
@@ -0,0 +1,269 @@
+But if prices slip too quickly due to a lack of liquidity, their profits are
+comparatively small to the value of their information.
+
+Thus, informants and liquidity providers are taking opposite sides of a bet.
+When liquidity providers sell to the informants, they are hoping that the assets
+are overpriced relative to their likelihood, while the traders assume that the
+asset is underpriced. But when informants buy, the prices go up, which means,
+provided the LPs are correct, that they are overpaying, resulting in a profit
+for the LPs. The same applies to selling.
+
+As an extreme example, imagine the informants are buying out a token which the
+LPs know to be incorrect, even at prices close to 1$. The LPs receive fees and
+keep the profits made from selling the incorrect outcome tokens, which will be
+worthless after the market has resolved. But they also kept the correct outcome
+tokens (their initial investment), giving them a considerable profit.
+Conversely, if there's an information shock and the informants are able to
+conclude which outcome the market will resolve to, they will buy all of these
+tokens from the pool and leave the LPs holding bags of incorrect (this is,
+worthless) outcome tokens.
+
+In particular, providing liquidity is just as risky as trading on a market. In
+case of an _information shock_, you stand to lose your entire position (for
+example, if the outcome of the event is revealed before the market closes). When
+the LPs take a loss to the informants, this may be explained by thinking of the
+liquidity providers' losses as payment for the informants' services.
+
+Increasing the liquidity of a pool by adding tokens does not change any of the
+prices, but it has the effect of making it harder for traders to move the
+prices. This is sometimes describes by saying that the market's thickness is
+increased. As a result, traders as able to take greater risks by opening larger
+positions, incentivizing them more to give up what they know about future
+events. But from the LPs perspective, this is also a good thing, as it gives the
+informants more rope. Thus, providing liquidity can be thought of increasing
+your position in your bet against the market.
+
+So, users should provide liquidity when the spot prices are at the level that
+they should be at according to their prediction. The traders then pay money to
+receive outcome tokens and shift the prediction to what they believe to be the
+correct probability distribution. If they get it right, the liquidity provider
+will incur a loss. But if the LPs initial prediction is correct, then their
+expected profit is non-negative (this is a hard calculation), and the farther
+the informants' probability distribution diverges from the LPs prediction and
+the more market noise that occurs while the price moves from the initial to the
+final prediction, the higher the LPs expected profit. Even if the final
```

Comment:
```suggestion
will incur a loss. But if the LPs initial prediction is correct, then their
expected profit is non-negative (this is a hard calculation), and the farther
the informants' probability distribution diverges from the LPs prediction and
the more market noise that occurs while the price moves from the initial to the
final prediction, the higher the LPs expected profit. Even if the final
```

Can you split this long sentence?

Reply:
Example:

    If the LPs initial prediction is correct, their expected profit is non-negative.
    Calculating this profit can be complex.
    The informants' probability distribution might diverge from the LPs prediction.
    Market noise can also occur.
    Such noise happens as the price transitions from its initial to final prediction.
    Both these factors can increase the LPs expected profit.

Reply:
5bfdd9be3c7003c218717965beed81b3b651f43b